include/class.format.php

```diff
@@ -503,6 +503,22 @@ function array_implode( $glue, $separator, $array ) {
         return implode( $separator, $string );
     }
 
+    function number($number) {
```

Comment:
Should be 

```php

function number($number, $locale=null) { 
```

Reply:
swear I committed that change..updating in a sec

Reply:
updated